src/synapse/Player.php

```diff
@@ -194,7 +194,9 @@ protected function processLogin(){
 			$pk->y = $this->y;
 			$pk->z = $this->z;
 			$pk->seed = -1;
-			$pk->dimension = $this->level->getDimension();
+			if($this->server->getName() != "PocketMine-MP"){
```

Comment:
Not many don't have these methods, in fact the only I can recall is steadfast, regardless, I believe I have a solution.